tools/c7n_gcp/c7n_gcp/filters/labels.py

```diff
@@ -78,6 +80,16 @@ def process(self, resources, event=None):
         self.tz = Time.get_tz(self.data.get('tz', 'utc'))
         return super(LabelActionFilter, self).process(resources, event)
 
+    def parse(self, v: str):
+        msg, remainder = v.split('-', 1)
```

Comment:
The msg could contain a hyphen. Would that be an issue?

Reply:
afaics it not really possible to handle the ambiguity the same way going right to left. afaics wrt to hyphen in message it wasn't possible before, and we can explicitly disallow that in validation.

Reply:
Take `resource_policy-set-public-access-prevention-2022_08_23__0_0` as an example.
1. rsplit('-',1) -> ['resource_policy-set-public-access-prevention', '2022_08_23__0_0']
2. endswith(op) -> 'resource_policy' and  'set-public-access-prevention'

Reply:
fair enough